packages/cli-types/src/ios.ts

```diff
@@ -6,6 +6,13 @@
  */
 export interface IOSProjectParams {
   project?: string;
+  /**
+   * @deprecated A podspec should always be at the root of a package and
+   *             have the name of the package. This property will be
+   *             removed in a future major version.
+   *
+   * @todo Log a warning when this is used.
```

Comment:
Have to follow-up on this in a future iteration.

Reply:
We'll need to identify libraries that follow a pattern of adding a Podspec into `ios/` dir and send PRs.